doc/man3/BIO_ctrl.pod

```diff
@@ -34,6 +34,8 @@ BIO_get_info_callback, BIO_set_info_callback, BIO_info_cb
  int BIO_get_info_callback(BIO *b, BIO_info_cb **cbp);
  int BIO_set_info_callback(BIO *b, BIO_info_cb *cb);
 
+ int BIO_get_ktls(BIO *b);
```

Comment:
Nit: Please add a HISTORY section in this man page for this macro (see other man pages for an example)

Reply:
Done.